en/SUMMARY.md

```diff
@@ -29,21 +29,32 @@
   * [Interfacing to ROS](simulation/ros_interface.md)
   * [AirSim Simulation](simulation/airsim.md)
   * [Multi-Vehicle Simulation](simulation/multi-vehicle-simulation.md)
-* [Tutorials](tutorials/tutorials.md)
-  * [Ground Control Station](qgc/README.md)
-  * [Writing an Application](tutorials/tutorial_hello_sky.md)
-  * [Video Streaming in QGC](qgc/video_streaming.md)
-  * [Long-distance Video Streaming](qgc/video_streaming_wifi_broadcast.md)
+* [Hardware](hardware/README.md)
+  * [Autopilot Hardware](flight_controller/README.md)
```

Comment:
[This](https://hamishwillee.gitbooks.io/havdevguide/content/v/1c01c94cbbd1bfcfabe658092ebdda87e1574ed1/en/flight_controller/) just points the the hardware ref in user guide. Would be useful to have other docs here on porting to new boards - is that what this [Porting guide](https://hamishwillee.gitbooks.io/havdevguide/content/v/1c01c94cbbd1bfcfabe658092ebdda87e1574ed1/en/debug/porting-guide.html) is for?

Reply:
Yes. It's a bit sparse though, but I haven't gone through the process of bringing up a new board myself.

Reply:
True :-(.  OK, action before merging is to better link porting guide and this doc.

Post process is to add more "meat to that guide". Do you know of anyone other than @LorenzMeier who has porting experience?